patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -262,7 +261,29 @@
           }
        } else {
           return null;
-@@ -2320,9 +2357,10 @@
+@@ -2123,14 +2160,19 @@
+       this.field_70128_L = true;
+    }
+ 
+-   @Nullable
++   @Nullable @Deprecated // Use ITeleporter safe version
+    protected PortalInfo func_241829_a(ServerWorld p_241829_1_) {
++      return this.func_241829_a(p_241829_1_, p_241829_1_.func_85176_s());
++   }
++
++   @Nullable
++   protected PortalInfo func_241829_a(ServerWorld p_241829_1_, net.minecraftforge.common.util.ITeleporter teleporter) {
```

Comment:
This method should have a proper name and be added to the `forge.exc` file so that the `ServerWorld` param can be properly remapped.

Reply:
Taken care of. The new method has been named `getPortalInfo` and exists within the `forge.exc` for it's `ServerWorld` param.